src/main/electron/listener.cljs

```diff
@@ -147,34 +151,39 @@
                      (fn [code]
                        (user/login-callback code)))
 
-  (js/window.apis.on "quickCapture"
-                     (fn [args]
-                       (let [{:keys [url title content]} (bean/->clj args)
-                             page (or (state/get-current-page)
-                                      (string/lower-case (date/journal-name)))
-                             format (db/get-page-format page)
-                             time (date/get-current-time)
-                             text (or (and content (not-empty (string/trim content))) "")
-                             link (if (not-empty title) (config/link-format format title url) url)
-                             template (get-in (state/get-config)
-                                              [:quick-capture-templates :text]
-                                              "**{time}** [[quick capture]]: {text} {url}")
-                             content (-> template
-                                         (string/replace "{time}" time)
-                                         (string/replace "{url}" link)
-                                         (string/replace "{text}" text))]
-                         (if (and (state/get-edit-block) (state/editing?))
-                           (editor-handler/insert content)
-                           (editor-handler/api-insert-new-block! content {:page page
-                                                                          :edit-block? false
-                                                                          :replace-empty-target? true})))))
 
   (js/window.apis.on "openNewWindowOfGraph"
                      ;; Handle open new window in renderer, until the destination graph doesn't rely on setting local storage
                      ;; No db cache persisting ensured. Should be handled by the caller
                      (fn [repo]
                        (ui-handler/open-new-window! repo))))
 
+
+(js/window.apis.on "quickCapture"
+                   (fn [args]
+                     (let [{:keys [url title content]} (bean/->clj args)
+                           insert-today? (get-in (state/get-config)
+                                                 [:quick-capture-options :insert-today]
+                                                 true)
+                           page (if (true? insert-today?) (string/lower-case (date/today)) (or (state/get-current-page) (string/lower-case date/journal-name)))
```

Comment:
What's the different between `(string/lower-case date/journal-name)` and `(string/lower-case (date/today))` ?

Reply:
You are right! I am not sure what I was thinking.